Detectors/Base/include/DetectorsBase/BaseHits.h

```diff
@@ -0,0 +1,67 @@
+#ifndef ALICEO2_BASE_HIT_H
+#define ALICEO2_BASE_HIT_H
+#include "FairMultiLinkedData_Interface.h"
+#include "Math/GenVector/DisplacementVector3D.h"
+#include "Math/GenVector/PositionVector3D.h"
+#include "Math/SVector.h"
+
+template <typename T>
+using Point3D = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::DefaultCoordinateSystemTag>;
+template <typename T>
+using Vector3D = ROOT::Math::DisplacementVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::DefaultCoordinateSystemTag>;
+
```

Comment:
I would advocate for a global definition of these types and consistent use in our interfaces.